### DISCUSSION TITLE
Well established architecture

### DISCUSSION THREAD

**Discussion by tomr6** [danielegrattarola/spektral#271] [General]
Can you form an example  architecture of gooGleNet or U-Net. I would like to test difference of GCN from CNN using well established architecture. or it is meaningless? and, is it possible to use TopKPool in batch-loader ?

**Comment by danielegrattarola** [comment]
Hi, 

there are deep GNN architectures that resemble typical CNN architectures (https://arxiv.org/abs/1905.05178), but it depends on what you are trying to show. 
If the task is image classification, then I think that the comparison between a CNN and GNN might not be that meaningful. 

>and, is it possible to use TopKPool in batch-loader ?

For now, no. The layer only supports disjoint mode. 

Cheers

**Reply by tomr6** [reply]
Dear developer. 
Thank you for the comment. Yes, I know this paper: Graph U-Net though I've not read it carefully. Though I'm not sure which pooling is useful for Graph U-Net, I expect that TopKPool will gets available in the future with Batch-mode  (Torch has a function of Graph U-Net with TopKPool). I'd like to test Graph U-Net for my work with the framework of Spektral. 
Best regards.
